.gitignore

```diff
@@ -23,3 +23,4 @@
 /.pydevproject
 /.idea
 /*.iml
+*.swp
```

Comment:
I don't understand why there are so many .gitignore files in this repo, especially since Git "inherits" ignores and most of them are exactly the same. Adding it here will make it ignored everywhere in the repo.